File: 283 Move Zeroes/283 Move Zeroes.js
var moveZeroes = function(nums){
    var i=0, c= nums.length;
    while(i < c)
        if (nums[i] == 0){
            nums.splice(i,1);
            nums.push(0);
        }else{
            i++;
        }
}
//Min:136ms

File: 283 Move Zeroes/283 Move Zeroes.c
void moveZeroes(int *nums, int mumsSize){
    int i, j;
    for(i = 0, j = 0; i<numsSize; i++)
        if(*(nums+i)!=0){
            *(nums+j) = *(nums+i);
            j++;
        }
    for(; j < numsSize; j++)
        *(nums+j) = 0
}
//8ms